### PR TITLE
"Include all online results" button for campus facet

### LINKED ISSUE #529: Add "add all Online items" to search results already faceted by Library
Users (and the library people helping them) at a particular location can see all items immediately available to them, which means everything which has a location in the library they're currently at and everything that's available Online, since they can immediately access that too. 

In terms of results, this means everything selected by the Library facet, e.g. York, AND everything that is Online. (Maybe some kind of special button up top? "Include all online results"?)

## WDDLL

- [ ] Users who are faceting to a single Library can also add _all_ items Online.

## Example link

Right now there is no way to make the inclusive selection. the ANY choice in Advanced options only applies to the top section. The controlled values are "AND have these attributes" and selecting both Online & Hershey only shows both: https://catalog.libraries.psu.edu/catalog?utf8=%E2%9C%93&op=AND&all_fields=&title=&author=&subject=&identifiers=&series=&publisher=&f_inclusive%5Baccess_facet%5D%5B%5D=Online&f_inclusive%5Blibrary_facet%5D%5B%5D=College+of+Medicine+%28Hershey%29&range%5Bpub_date_itsi%5D%5Bbegin%5D=&range%5Bpub_date_itsi%5D%5Bend%5D=&sort=score+desc%2C+pub_date_itsi+desc%2C+title_sort+asc&search_field=advanced&commit=Search

### PR DESCRIPTION
**Pull request by ajkiessl**
The include button 👇 
Screenshot 2023-04-17 at 4 31 56 PM

And the remove button 👇 
Screenshot 2023-04-17 at 4 33 17 PM

There's no logic to remove the 'add_all_online' query parameter in the URL after clicking the include button when the user leaves the single campus facet search.  However, there is logic to ignore this parameter when the user is not in a single campus facet search.  Therefore, this should not cause any confusion or issues unless the users are very picky about URLs.

closes #529

### CODE REVIEW

File: app/helpers/facets_helper.rb [hunk 1]
```diff
@@ -60,4 +60,22 @@ def facet_item_component_class(facet_config)
     default_component = facet_config.pivot ? Blacklight::FacetItemPivotComponent : PsulFacetItemComponent
     facet_config.fetch(:item_component, default_component)
   end
+
+  def campus_facet_all_online_links
```
Comment (ajkiessl) [inline app/helpers/facets_helper.rb:64]: Also, I wasn't too sure if this was the right place to put this method.  It's related to the facets, but isn't actually in the facets drop-down.

**Comment by ruthtillman** [comment]
@ajkiessl I really like that button! How does it look on mobile?

**Comment by ajkiessl** [comment]
@ruthtillman 
Screenshot 2023-04-18 at 10 11 03 AM
Screenshot 2023-04-18 at 10 11 16 AM

File: app/helpers/facets_helper.rb (hunk 1 above)
Comment (banukutlu) [reply app/helpers/facets_helper.rb:64]: I think it makes sense here 👍🏻

**Review by banukutlu — APPROVED** [review]
looks great! thanks Alex :shipit:

**Pull request merged**